src/python_testing/matter_testing_infrastructure/chip/testing/matter_testing.py

```diff
@@ -292,14 +198,18 @@ def __call__(self, res: EventReadResult, transaction: SubscriptionTransaction):
     def wait_for_event_report(self, expected_event: ClusterObjects.ClusterEvent, timeout_sec: float = 10.0) -> Any:
         """This function allows a test script to block waiting for the specific event to be the next event
            to arrive within a timeout (specified in seconds). It returns the event data so that the values can be checked."""
-        logging.info(f"Waiting for {expected_event} for {timeout_sec:.1f} seconds")
+        logging.info(
```

Comment:
It looks like there's a bunch of formatting changes that got added in here - would you mind putting these back? It's making it a bit hard to tell what's actually changed.

Reply:
Reverted